lib/class-wp-style-engine-gutenberg.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * WP_Style_Engine class
+ *
+ * @package Gutenberg
+ */
```

Comment:
I'll focus my review on this file, as If I'm not wrong, this is the "style engine" package equivalent. Well I guess my first question is, should we try to move this file to the package (like we do for "parser", and "block-library")?

Reply:
> should we try to move this file to the package (like we do for "parser", and "block-library")

Good question! I suppose it depends on what will be easiest for when the feature needs to be merged into core. I see Ramon asked a similar question over on: https://github.com/WordPress/gutenberg/pull/39446#issuecomment-1075720655 — given that PR will be in better shape to land before this one, it might be better for us to explore where the file goes in that PR. For the moment, I mostly just put it here for convenience while hacking around!